src/h5web/explorer/Explorer.test.tsx

```diff
@@ -1,15 +1,15 @@
 import { fireEvent, screen } from '@testing-library/react';
-import { MOCK_DOMAIN } from '../providers/mock/MockProvider';
+import { mockDomain } from '../providers/mock/data';
```

Comment:
I need the mock domain when generating the nested mock data (it's the `name` of the root group), so I moved it to `data.ts`.